docs/_quarto.yml

```
@@ -160,6 +165,30 @@ quartodoc:
           contents:
             - ScaleStandard
 
+        - kind: page
+          path: steps-discretization
+          summary:
+            name: Discretization
+            desc: Discretization of numeric columns
+          contents:
+            - DiscretizeKBins
+
+        - kind: page
+          path: steps-feature-selection
+          summary:
+            name: Feature Selection
+            desc: Selecting useful features for modeling
+          contents:
+            - ZeroVariance
+
+        - kind: page
+          path: steps-feature-engineering
+          summary:
+            name: Feature Engineering
```

```suggestion
            name: Feature selection
            desc: Selecting useful features for modeling
          contents:
            - ZeroVariance

        - kind: page
          path: steps-feature-engineering
          summary:
            name: Feature engineering
```
Nit: I was going to ask about the case here, but it seems Quarto automatically standardizes the case, based on your screenshot? Think it looks good overall for now.